bench/qemu-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
I don't like that there are two scripts for running qemu (run-qemu.sh and /bench/qemu-test.sh) that have a lot of overlap. Is it possible to merge these into a single script?

Reply:
yes. I think I can rework it

Reply:
run-qemu.sh is now deleted. sudo ./qemu-test.sh (with no CL args) should do the same thing as run-qemu